docs/how-to/version-statuses.md

```diff
@@ -0,0 +1,30 @@
+
+
+## Route Information
+Navigate to: \
+```/org/{:organization}/{:project}/{:application}/versions```
+
+## Verification Steps
+Locate the Version List Table. Version statuses are displayed in the Status column.
+
+### Updating Version Status
+Click the Actions button to open the dropdown menu.
```

Comment:
This is closer to being on the right track, however there is no description on what these are and what they do in a meaningful manner to a end user.